### DISCUSSION TITLE
Issue installing extension

### DISCUSSION THREAD

**Discussion by martinwachira** [community/community#7300] [Projects and Issues] [answered]
command 'github.copilot.generate' not found

How do I resolve this?

**Comment by davecheney** [comment, marked as answer]
Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by martinwachira** [reply]
Yes, I was on a vpn. Fortunately I was able to install on another machine.

**Reply by martinwachira** [reply]
How would I get to install it on a firewall or a vpn?
